dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -29,33 +29,35 @@
                 :name="category.name"
               >
                 <div class="uploadFormSection grid" v-for="subcategory in category.subcategories" :key="subcategory">
-                  <div class="col-3 p-3 topicLabel">
-                    <h4 :id="subcategory.name" class="anchor title">{{ subcategory.label }}</h4>
-                    <div :class="`p-badge badge-${category.color}`">
-                      <span>{{ category.label.toUpperCase() }}</span>
+                  <template v-if="isSubcategoryVisible.get(subcategory) ?? true">
```

Comment:
Just out of curiousity: The logic which decides if a subcategory is shown has been put into a "v-if" as method here, to dynamically do the check and adjust to changes in the form, right?
That's the reason the map cannot be built only once at the beginning.

Reply:
Yes. The map must respond to changes in the form.

Reply:
👍